agent/wsclient/client.go

```diff
@@ -132,6 +132,7 @@ type ClientServerImpl struct {
 // 'MakeRequest' can be made after calling this, but responss will not be
 // receivable until 'Serve' is also called.
 func (cs *ClientServerImpl) Connect() error {
+	seelog.Debugf("Establishing a Websocket connection to %s", cs.URL)
```

Comment:
With the log here, I think we should delete this [line](https://github.com/aws/amazon-ecs-agent/blob/dev/agent/tcs/handler/handler.go#L92), they are duplicate.

Reply:
I think I got all the duplicate lines [here](https://github.com/aws/amazon-ecs-agent/pull/964/files#diff-19e06c1031fd70ac815de965c115b519L92) but if not, I'll go find the others.